lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -109,6 +127,16 @@ def get(self, request, *args, **kwargs):
         is_enrolled = enrollment and enrollment.is_active
         is_staff = has_access(request.user, 'staff', course_key)
 
+        course_outline_root_block = get_course_outline_block_tree(request, course_key_string, request.user)
+        resume_block = get_resume_block(course_outline_root_block) if course_outline_root_block else None
+        has_visited_course = bool(resume_block)
+        if resume_block:
+            resume_course_url = resume_block['lms_web_url']
+            resume_course_title = resume_block['display_name']
+        else:
+            resume_course_url = course_outline_root_block['lms_web_url'] if course_outline_root_block else None
+            resume_course_title = course_outline_root_block['display_name'] if course_outline_root_block else None
```

Comment:
Maybe a `elif course_outline_root_block:` with a `else:` for the None cases might make these three cases more clear?